src/form-elements/FormElementSignature.tsx

```diff
@@ -190,6 +191,7 @@ const SignatureDisplay = React.memo(function SignatureDisplay({
   value: Props['value']
   onChange: Props['onChange']
 }) {
+  const onUploadAttachment = useOnUploadAttachmentContext()
```

Comment:
move this to the `useAttachment` hook as you dont need to pass this in if its the same every time.